File: 1534. Count Good Triplets/solution.cpp
class Solution {
public:
  int countGoodTriplets(vector<int>& arr, int a, int b, int c) {
    int sizeOfArr = arr.size();
    int count = 0;
    for (int i = 0; i < sizeOfArr - 2; i++){
      for (int j = i + 1; j < sizeOfArr - 1; j++){
        for (int k = j + 1; k < sizeOfArr; k++){
          if (abs(arr[i] - arr[j]) <= a && abs(arr[j] - arr[k]) <= b && abs(arr[i] - arr[k]) <= c){
            count += 1;
          }
        }
      }
    }
    return count;
  }
};

File: 1534. Count Good Triplets/solution.py
class Solution(object):
  def countGoodTriplets(self, arr, a, b, c):
    """
    :type arr: List[int]
    :type a: int
    :type b: int
    :type c: int
    :rtype: int
    """
    sizeOfArr = len(arr)
    count = 0
    for i in range(sizeOfArr - 2):
      for j in range(i + 1, sizeOfArr - 1):
        for k in range(j + 1, sizeOfArr):
          if (abs(arr[i] - arr[j]) <= a and abs(arr[j] - arr[k]) <= b and abs(arr[i] - arr[k]) <= c):
            count += 1
    return count